LayoutTests/platform/ios-wk2/TestExpectations

```diff
@@ -2766,6 +2766,6 @@ http/tests/security/clipboard/copy-paste-html-across-origin-sanitizes-html.html
 http/tests/security/clipboard/copy-paste-html-across-origin-strips-mso-list.html [ Pass Failure ]
 imported/w3c/web-platform-tests/content-security-policy/script-src/script-src-report-only-policy-works-with-external-hash-policy.html [ Pass Failure ]
 
-webkit.org/b/271442 fast/events/ios/rotation/basic-rotation.html [ Failure ]
+fast/events/ios/rotation/basic-rotation.html [ Pass ]
```

Comment:
I think we can remove the expectation entirely?

Reply:
I had to keep the expectation because that entire subdirectory is skipped in `platform/ios/TestExpectations`

https://github.com/WebKit/WebKit/blob/85534d1cfd22306695c4869c541f33f5940bf077/LayoutTests/platform/ios/TestExpectations#L4461

Reply:
That's unfortunate 😞 

I wonder how many tests are actually still failing. Otherwise we're just going to have `[ Pass ]` results littered around for tests in that directory, and top-level `[ Skip ]` will always remain...

Reply:
That's a good point. Let me remove that expectation for now and verify...

Reply:
The expectations are stale, and I can't reproduce the timeout prompting these expectations. I'll remove it.